### PR TITLE
Migrate telemetry to stdlib logging module & logz.io

### PR DESCRIPTION
**Pull request by cmyui**

### CODE REVIEW

File: common/generalUtils.py [hunk 1]
```diff
@@ -76,13 +77,13 @@ def fileMd5(filename: str) -> str:
...
-    c100: Optional[int] = None,
-    c50: Optional[int] = None,
-    cmiss: Optional[int] = None,
+    gameMode: int,
+    __mods: int,
+    acc: float,
+    c300: int,
+    c100: int,
+    c50: int,
+    cmiss: int,
```
Comment (cmyui) [inline common/generalUtils.py:85]: unrelated to this pr - just typing simplification

File: common/log/rap_logs.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+from common.ripple import userUtils
+from common.web.discord import Webhook
+from objects import glob
+
+THREAD_POOL = ThreadPoolExecutor(max_workers=4)
+
+ENDL = "\n" if name == "posix" else "\r\n"
+
+RETRY_INTERVAL = 1
+MAX_RETRIES = 3
```
Comment (cmyui) [inline common/log/rap_logs.py]: now that the firing of discord webhooks is done in the background, we can enforce a much more aggressive retry policy
```suggestion
RETRY_INTERVAL = 8
MAX_RETRIES = 10
```

File: constants/fokabotCommands.py [hunk 3]
```diff
@@ -600,11 +601,20 @@ def getPPMessage(userID: int, just_data: bool = False) -> Any:
...
-    except Exception as e:
-        print(e)
+    except Exception as exc:
+        logger.error(
+            "Failed to retrieve PP from LESS API",
+            exc_info=exc,
+            extra={
+                "userID": userID,
+                "currentMap": currentMap,
+                "currentMods": currentMods,
```
Comment (cmyui) [inline constants/fokabotCommands.py]:
```suggestion
                "user_id": userID,
                "beatmap_id": currentMap,
                "mods": currentMods,
```

**Review by tsunyoku — DISMISSED** [review]

**Comment by tsunyoku** [comment]
gl with conflicts XD

**Comment by cmyui** [comment]
> gl with conflicts XD

image

**Comment by cmyui** [comment]
Resolving conflicts will be harder than remaking it - will salvage what I can into a new branch.

**Pull request closed without merging**